Gemfile

```diff
@@ -28,4 +28,9 @@ group :development do
   gem "simplecov-lcov", "~> 0.8"
   gem "standard"
   gem "timecop", "~> 0.9"
+
+  # Rails dummy and ammeter dependencies
+  gem "activerecord"
+  gem "ammeter"
+  gem "pg"
```

Comment:
Does it make sense to use the dummy app without active record to avoid having these deps at all?